src/test/java/com/epam/service/MockPoliceForceServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.service;
+
+import com.epam.service.MockPoliceForceService;
+import org.junit.Test;
+
+import java.util.concurrent.*;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNot.not;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Created by Alexey on 03.05.2016.
+ */
+public class MockPoliceForceServiceTest {
```

Comment:
Well, I made a confusion here naming the class with Mock... prefix. It's actually dummy service emulating network communication. So there's no reason to mock the service here as I'm testing real dummy service. Renamed the service now.

From other side this is not unit test at all but integration one:)